src/wombats/sockets/game.clj

```diff
@@ -157,11 +188,23 @@
   [datomic]
   (fn [{:keys [chan-id :user/id] :as socket-user}
       {:keys [game-id]}]
-    (let [player ((:get-player-from-game datomic) game-id id)]
+    (let [player ((:get-player-from-game datomic) game-id id)
+          game ((:get-game-by-id datomic) game-id)
+          game-state ((:get-game-state-by-id datomic) game-id)
+          arena (get-in game-state [:frame :frame/arena])]
+
       (swap! game-rooms
              assoc-in
              [game-id :players chan-id]
-             (assoc socket-user :color (:player/color player))))))
+             (assoc socket-user :color (:player/color player)))
+
+      ;; Sends the initial frame to the frontend (TODO: do we really need these separated though?)
+      (broadcast-arena game-id arena)
```

Comment:
We probably will want to skip the arena/player broadcasts if the game is in progress. Would like to talk through this tomorrow.